translations/en-us.json

```diff
@@ -4,6 +4,8 @@
   "brand_color_label": "Brand color",
   "brand_text_color_description": "Brand color for hover and active states",
   "brand_text_color_label": "Brand text color",
+  "brand_name_label": "Show Help Center name",
+  "brand_name_description": "Display the Help Center name next to the logo",
```

Comment:
The `translations/*.json` files are generated automatically, we should not update them manually. We can instead run `node bin/update-translations.js` in the project root. That will download all translations and add them to the json files :)

Reply:
Will fix this. I don't know why it did this. Think it was a merge conflict that I fixed wrong then.